http3/multipart.py

```diff
@@ -83,9 +89,11 @@ def iter_fields(data: dict, files: dict) -> typing.Iterator[Field]:
         yield FileField(name=name, value=value)
 
 
-def multipart_encode(data: dict, files: dict) -> typing.Tuple[bytes, str]:
+def multipart_encode(
+    data: dict, files: dict, boundary: bytes = None
```

Comment:
imo we shouldn't expose the boundary interface publicly. What use-case is there to allowing users to control it?

Reply:
only reason was to make it easier to test. similar to urlib3. if it is unacceptable it can be removed.

Reply:
Can be tested by mocking out `os.urandom()`. I think it should be removed as it shouldn't be a necessary interface.